src/DynamoCore/Core/Threading/UpdateRenderPackageAsyncTask.cs

```diff
@@ -183,17 +187,59 @@ private void GetRenderPackagesFromMirrorData(MirrorData mirrorData, ref List<str
                         }
                     }
 
+                    var plane = graphicItem as Plane;
+                    if (plane != null)
```

Comment:
The new representation planes seems a bit light. Part of the reason I made the planes look the way they do, is that they don't look like any other geometry type. These look just like patched surfaces. But this is probably just a matter of taste.

![plane](https://cloud.githubusercontent.com/assets/289141/7209536/4e8022b8-e4fe-11e4-918d-648c6c5c6332.png)